internal/cmd/util/util.go

```diff
@@ -377,3 +378,29 @@ func IterateInOrder[M ~map[K]V, K cmp.Ordered, V any](m M) iter.Seq2[K, V] {
 		}
 	}
 }
+
+func FormatHcloudError(err error) string {
+	if err == nil {
+		return ""
+	}
+
+	var hcloudErr hcloud.Error
+	if !errors.As(err, &hcloudErr) {
+		return err.Error()
+	}
+
+	switch hcloudErr.Code {
+	case hcloud.ErrorCodeInvalidInput:
+		details := hcloudErr.Details.(hcloud.ErrorDetailsInvalidInput)
+		var errBuilder strings.Builder
+		errBuilder.WriteString(hcloudErr.Message)
```

Comment:
If we want to include the correlation ID we can still use `hcloudErr.Error()` here and add the details at the end.

```suggestion
		errBuilder.WriteString(hcloudErr.Error())
```